packages/cozy-client/src/helpers/urlHelper.js

```diff
@@ -44,7 +44,7 @@ export const generateWebLink = ({
           .split('.')
           .map((x, i) => (i === 0 ? x + '-' + slug : x))
           .join('.')
-  url.pathname = pathname
+  url.pathname = pathname || ''
```

Comment:
shouldn't we use `ensureFirstSlash` as just below for the `hash`?

Reply:
The result would be the same, `URL` will internally convert `''` to `'/'`

Do you prefer the ensureFirstSlash version?

Reply:
to be consistent with the line just below your code, yep. Also `/` speaks to me for a path while `''`  doesn't.

Reply:
Done, I also updated the commit and PR messages

Reply:
I'll merge just after I get the CI result